src/app/applications/application-add-edit/application-add-edit.component.html

```diff
@@ -2,7 +2,7 @@
   <div class="container">
     <div class="title-container">
       <div class="title-container__title">
-        <h1><span class="text-muted">Crown land File: {{application['clFile']}} &nbsp;&rsaquo;&nbsp; </span>
+        <h1><span class="text-muted">Crown land File: {{application.clFile}} &nbsp;&rsaquo;&nbsp; </span>
```

Comment:
Any reason to use bracket notation over dot?

Reply:
If the property is used in multiple places (components) then I used dot notation to make the property part of the model and use the power of TypeScript (for type checking). If the property is used "temporarily" in a single component then I used bracket notation to make the property dynamic.
This is a self-enforced convention.